tests/blueprints/v1/test_cherrypicker_data_blueprint.py

```diff
@@ -4,21 +4,30 @@
 
 import pytest
 
-from crawler.constants import FIELD_STATUS_COMPLETED, FLASK_ERROR_MISSING_PARAMETERS, FLASK_ERROR_UNEXPECTED
+from crawler.constants import (
+    FIELD_STATUS_COMPLETED,
+    FLASK_ERROR_MISSING_PARAMETERS,
+    FLASK_ERROR_UNEXPECTED,
+)
```

Comment:
What's the reason behind this change?

Reply:
I didn't explicitly make this change.  It's part of the reformat on save that you said I should add to VSCode.  Even if I  change it back, it goes back to this on save.

Reply:
Hmm something funny is happening, when I save that file it keeps the imports on one line since it does not go over the 120 `line-length` in `pyproject.toml`. Do you have `black` set up as your default formatter for python?

Reply:
I've updated the import sorting parameters in my VSCode like so.  This stops it and `black` from fighting over the style:

```
    "python.sortImports.args": [
        "--multi-line=3",
        "--trailing-comma",
        "--line-length=120"
    ]
```